invenio_records_resources/services/records/service.py

```diff
@@ -357,6 +358,15 @@ def read(self, identity, id_, expand=False):
             expand=expand,
         )
 
+    def exists(self, identity, id_):
+        """Check if the community exists exists."""
+        try:
+            record = self.record_cls.pid.resolve(id_)
+            self.require_permission(identity, "read", record=record)
+        except PIDDoesNotExistError:
+            return False
+        return True
```

Comment:
I think it would be safer to return True inside the try block, since we want to return True only if there are no exceptions, WDYT?

Reply:
Your right!
I'll edit thx